### PR TITLE
flywheel/pydeface-gear:1.0.0

### PR DESCRIPTION
**Pull request by joshicola**
First customer distribution.

### CODE REVIEW

File: gears/flywheel/pydeface-gear.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
+{
+  "name": "pydeface-gear",
+  "label": "Pydeface Gear",
+  "description": "A gear to remove facial structure from MRI images.",
+  "author": "Flywheel",
```
Comment (lmperry) [inline gears/flywheel/pydeface-gear.json]: poldracklab?

File: gears/flywheel/pydeface-gear.json [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  "maintainer": "Flywheel <support@flywheel.io>",
+  "license": "MIT",
+  "url": "https://github.com/flywheel-apps/test-gear",
```
Comment (lmperry) [inline gears/flywheel/pydeface-gear.json]: https://github.com/flywheel-apps/pydeface-gear?

File: gears/flywheel/pydeface-gear.json (hunk 1 above)
Comment (joshicola) [reply gears/flywheel/pydeface-gear.json]: Will they be maintaining this gear?

Comment (lmperry) [reply gears/flywheel/pydeface-gear.json]: No.

**Pull request merged**